File: Beecrowd/Java/1113 - Ascending and Descending.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int X,Y;
        while(true){
            X=sc.nextInt();
            Y=sc.nextInt();
            if(X==Y)break;
            if(X>Y){
                System.out.println("Decrescente");
            }
            else System.out.println("Crescente");
        }
    }
}

File: Beecrowd/C++/1113 - Ascending and Descending.cpp
#include<iostream>
int main(){
    int X,Y;
    while(1){
    std::cin>>X>>Y;
    if(X==Y)break;
    if(X>Y){
        std::cout<<"Decrescente"<<std::endl;
    }
    else std::cout<<"Crescente"<<std::endl;
    }
}
